lib/remote/endpoint.ti

```diff
@@ -10,6 +10,8 @@ namespace icinga
 
 class Endpoint : ConfigObject
 {
+	activation_priority 300;
+
```

Comment:
What is this for? Neither Endpoint overrides #Start() or #Stop(), nor any code calls Endpoint#IsActive().

Reply:
Because I did not notice till rn, that all communication with the endpoints also counts as ApiListener.... I expected them to behave like the other config objects... My brain hurts now.